src/commcare_cloud/ansible/deploy_control.yml

```diff
@@ -1,6 +1,5 @@
 - name: Bootstrap Control
-  hosts: localhost
-  gather_facts: no
+  hosts: control
```

Comment:
won't this make it ssh into the control machine? or is that the point so that you can run it from elsewhere?

Reply:
Yeah that's the point. I wanted to be able to use this to bootstrap a control machine remotely. It should also still work from the control machine itself, it'll just be sshing into itself.